motioneye/handlers/action.py

```diff
@@ -66,6 +66,16 @@ async def post(self, camera_id, action):
             )
             return self.record_stop(camera_id)
 
+        elif action == 'eventstart':
+            logging.debug(f'executing event_start action for camera with id {camera_id}')
```

Comment:
To satisfy pre-commit (max line length):
```suggestion
            logging.debug(
                f'executing event_start action for camera with id {camera_id}'
            )
```